src/lib/components/Grid/index.js

```diff
@@ -814,13 +820,19 @@ const GridBase = memo(({
         setSortModel(sort);
     }
 
-    const breadCrumbs = [
-        { text: model.gridTitle }
-    ]
+    let breadCrumbs;
+
+    if (model?.searchParamKey) {
```

Comment:
Pls destructure at top of function so this can be widely use with default values